lib/bolt/cli.rb

```diff
@@ -861,17 +866,35 @@ def install_puppetfile(config, puppetfile, moduledir)
     # modules being configured.
     #
     def assert_puppetfile_or_module_command(modules)
+      if Bolt::Util.powershell?
+        case options[:action]
+        when 'generate-types'
+          old_command = 'Register-BoltPuppetfileTypes'
+          new_command = 'Register-BoltModuleTypes'
+        when 'install'
+          old_command = 'Install-BoltPuppetfile'
+          new_command = 'Install-BoltModule'
+        when 'show-modules'
+          old_command = 'Get-BoltPuppetfileModules'
+          new_command = 'Get-BoltModule'
+        end
```

Comment:
This also needs a clause for 'add' and 'show' if you're using the `bolt module` command without `modules` in config.